glean-core/ios/Glean/Scheduler/GleanLifecycleObserver.swift

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+class GleanLifecycleObserver {
```

Comment:
nit: should we have docs for this? Will `GleanLifecycleObserver` be part of the API surface?

Reply:
I don't think it should show up as part of the public API, according to [documentation](https://docs.swift.org/swift-book/LanguageGuide/AccessControl.html) the default access in Swift is internal. You have to be explicit about public access, so it was not my intent to allow this to leak to the public API.  (Unless you see a need to, that is)

Reply:
If you think this should be public, we can file a follow up bug to this.